vulkan/vulkan_application.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_VULKAN_VULKAN_APPLICATION_H_
+#define FLUTTER_VULKAN_VULKAN_APPLICATION_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+#include "flutter/vulkan/vulkan_debug_report.h"
+#include "flutter/vulkan/vulkan_handle.h"
+#include "lib/ftl/macros.h"
+
+namespace vulkan {
+
+class VulkanDevice;
+class VulkanProcTable;
+
+class VulkanApplication {
```

Comment:
> Class comment please.

Done.

> I can't find any code in this CL that instantiates a VulkanApplication. I presume there must be, because otherwise one wouldn't be able to instantiate a VulkanSurface.
> Actually, I also can't find any code that instantiates a VulkanSurface. Am I using ctrl-F wrong? Since the no-arg VulkanSurface constructor is deleted in this CL, and a new one added, I would expect to see changes at the instantiation point.

Is your editor ignoring `VulkanWindow` for some reason? Both are instantiated there.